packages/react/.storybook/_container.scss

```diff
@@ -1,9 +1,88 @@
 $css--font-face: true;
+$css--helpers: true;
+$css--body: true;
+$css--use-layer: true;
 $css--reset: true;
+$css--plex: true;
 
 $prefix: 'bx';
 
-// IMPORTANT: This import path should _not_ be used outside our source tree
-// as `src` directory is _not_ meant to be shipped in our NPM package.
-// Use e.g. `@import '~carbon-components/scss/globals/scss/styles.scss'` instead.
-@import '~carbon-components/src/globals/scss/styles.scss'; // SEE THE NOTE ABOVE
+@import '~carbon-components/src/globals/scss/feature-flags';
+@import '~carbon-components/src/globals/scss/vars';
+@import '~carbon-components/src/globals/scss/colors';
+@import '~carbon-components/src/globals/scss/theme';
+@import '~carbon-components/src/globals/scss/mixins';
+@import '~carbon-components/src/globals/scss/layout';
+@import '~carbon-components/src/globals/scss/layer';
+@import '~carbon-components/src/globals/scss/spacing';
+@import '~carbon-components/src/globals/scss/typography';
+@import '~carbon-components/src/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
+@import '~carbon-components/src/globals/scss/css--reset';
+@import '~carbon-components/src/globals/scss/css--font-face';
+@import '~carbon-components/src/globals/scss/css--helpers';
+@import '~carbon-components/src/globals/scss/css--body';
+@import '~carbon-components/src/globals/grid/grid';
+
+$accordion-type-tokens: map-merge(
+  $tokens,
+  (
+    'body-long-01': $body-long-02,
```

Comment:
I'm really struggling with overriding `body-long-01`. As I mentioned last week when you were discussing this option with me, from my point of view the intent behind these tokens changes when we override it. As mentioned in the pull request this wouldn't allow there to be a 14px and 16px within the same component. You also now have two different tokens set to the same exact value.

I fear the bigger issue here is how we communicate this to our adopters and contributors. It sounds like our whole design kit will have to change. The inconsistency is also wild where `$body-long-01` for one component doesn't look the same for `$body-long-01` in another component. When someone says `$body-long-01` which value are we talking about now?

Reply:
Similar to color, we want to make sure that the tokens are being applied consistently across a global theme or across a zone. The ideal here is that `body-long-01` corresponds to specific moments of type usage, similar to how `interactive-01` is used. Here's the table on our site that shows some role examples for color tokens:

![image](https://user-images.githubusercontent.com/3901764/62075454-b655bc80-b20a-11e9-835b-4aa2af38a89a.png)


In the case of changing things within a component, this is where the zone theming idea comes in. We change the values of tokens for a given component context, but not for others. In this case, that would mean that these tokens get updated to their editorial theme equivalents in components that need to be in an editorial zone (like accordion).

If we look at tokens for the values they represent, then it's similar to just inlining the value directly. Instead, we should choose tokens for the design decisions they represent. 

Just a quick aside, this is definitely our first pass at tokens for type. If they feel off, we can definitely look into how we are applying these tokens and potentially updating them in places where their usage is not consistent.

Reply:
@joshblack I'm struggling to see how replacing `$interactive-02` is similar to replacing `$body-long-01`, and to be fair – I've told Matt this too – I could be way off on my understanding of type tokens.

It's my understanding that the type tokens are more elemental in nature, and ultimately part of the IBM Design Language. I saw it as almost a 1:1 tokenization of the IDL type sets. Similar to how you've tokenized the IDL colors. These are such elemental values that you don't want to change, right? It's how we communicate IBM's point of view of visual styling with one another. I look at setting `$body-long-01` to `$body-long-02` as changing its intent, and erasing a part of the IDL, and watering down how we communicate with each other.

With your color theming work you're not changing `$ibm-color__blue-50` to `$ibm-color__red-60`. It looks more like you've extracted that step out and defined specific meaning to those tokens like `$ui-background` is for UI Background and assigned IDL color value to it. If `$interactive-01` was set to `$ui-background` that would muddy the water. Just because their value is the same now doesn't mean their intent is, and thus that intent could direct a different value down the road.

If you want every component to use the same default type-token I don't personally see how `$body-long-01` takes on that intent. From my perspective, there should be a token like `$type-style-default` that extracts it a level and adds the intent y'all are discussing similarly to how the color theming works. That way your default might be set to `$body-long-01` out of the box, but dotcom it might be set to `$body-long-02`. Both `$body-long-01` and `$body-long-02` would maintain its original IDL values.

I hope that makes sense. Just something to think about I guess, so take it for what it is. 🤷‍♂️

Reply:
> It's my understanding that the type tokens are more elemental in nature, and ultimately part of the IBM Design Language. I saw it as almost a 1:1 tokenization of the IDL type sets.

Ah, this could definitely be where the problem is. Similar to colors, type tokens also have roles: https://www.carbondesignsystem.com/guidelines/typography/productive as I said earlier, if one is being used in error we can definitely fix it 👍 this is our first attempt at this.

From the IDL standpoint, we're definitely blurring things here as the two teams are getting closer and closer. The IDL will define our type face, IBM Plex, along with things like:

- Type scale
- Font weights
- How to pair font sizes with line heights
- How to pair font sizes with letter spacing

But when it comes to the design system, our intent is not to create a set of type tokens and map them to color variables like we have for `carbon--blue-50`. Things like the type scale, font weights, etc. do have corresponding variables that would mirror these parts of type. However, as we approach tokens, these do have role-based usage exactly like our color tokens.

Put another way:

Variables corresponding to the IDL:
- `carbon--blue-60` for color
- `carbon--type-scale` for type scale
- `carbon--font-weights` for font weights
- `carbon--font-families` for font families

Tokens corresponding to how we combine the above variables in the system:
- `interactive-01`: uses blue-60 under the hood, applied to interactive elements
- `body-long-01`: uses a combination of type scale, weight, font family, letter spacing, and line height under the hood. Used for 4 lines of text or more. 

It's important that things like `body-long-01` are **not** coming from the IDL, otherwise there would be no way for teams outside of IBM to theme the type styles of carbon with a font face different than IBM Plex

Reply:
Sounds good! Sorry about the confusion here @photodow, let me know what we could write about/document that would help prevent this in the future! 🙏